packages/react/src/components/LeadspaceWithSearch/README.stories.mdx

```diff
@@ -0,0 +1,81 @@
+import { Props, Description } from '@storybook/addon-docs/blocks';
+import contributing from '../../../../../docs/contributing-license.md';
+import LeadSpaceWithSearch from './LeadspaceWithSearch';
+
+# Leadspace with search
+
+> The Lead Space pattern is to be utilized within IBM.com.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+##### CSS
+
+```css
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+
+@import '@carbon/ibmdotcom-styles/scss/components/leadspace-with-search';
+```
+
+> 💡 Only import fonts once per usage. Don't forget to import the Leadspace
+> styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/master/packages/styles).
+
+##### JS
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { LeadspaceWithSearch } from '@carbon/ibmdotcom-react';
+import 'yourapplication.scss';
+
+function App() {
+  return (
+    <LeadSpaceWithSearch
+      heading="Find a product"
+      copy="Duis aute irure dolor in reprehen deritirure"
+      searchProps={{
+        onChange: (event) => {
+          // Now you can access the user inputed value under event.target.value
+          return null;
+        },
+        onKeyDown: (event) => {
+          // Now you can access the user inputed value under event.target.value
+          return null;
+        }
+      }}
+    />
+  );
+}
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+Add the following line in your `.env` file at the root of your project.
+[See more details](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/tree/master/packages/styles#usage).
+
+```
+  SASS_PATH=node_modules:src
+```
+
+> 💡 Don't forget to import the Leadspace With Search styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/master/packages/styles).
+
+## Props
+
+<Props of={LeadSpaceWithSearch} />
+
+## Stable selectors
+
+| Name                        | Description                      |
+| --------------------------- | -------------------------------- |
+| `bx--leadspace-with-search` | Pattern (grid container)         |
+| `dds--leadspace__row`       | LeadSpace grid row               |
+| `dds--leadspace__content`   | LeadSpace content (grid column)  |
+| `dds--leadspace__heading`   | Leadspace heading                |
+| `dds--leadspace__copy`      | LeadSpace optional copy          |
+| `dds--leadspace__search`    | LeadSpace search input           |
```

Comment:
wondering if this should have different stable selectors from the regular leadspace...